packages/metal-component/src/events/events.js

```diff
@@ -58,9 +58,9 @@ export function getComponentFn(component, fnName) {
 	if (isFunction(component[fnName])) {
 		return component[fnName].bind(component);
 	} else {
-		console.error('No function named "' + fnName + '" was found in the ' +
-			'component "' + getFunctionName(component.constructor) + '". Make ' +
-			'sure that you specify valid function names when adding inline listeners.'
+		console.error(`No function named ${fnName} was founc in the component
```

Comment:
There were a couple of minor errors here when converting to the new format:
* `found` became `founc`
* `valid` became `validf`

These are the only ones I found though, so I can just fix it myself and merge everything now :)